catalogs/qhimm.xml

```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
     </Mod>
 
   </Mods>
+
+## WereXIII - by Wolfman
+    <Mod>
+      <ID>1iwfYEec-eu6K4hB6zD06dok1jH92lX3n</ID>
+      <Author>Tsunamods</Author>
+      <Link>https://drive.google.com/file/d/1iwfYEec-eu6K4hB6zD06dok1jH92lX3n/view?usp=drive_link</Link>
```

Comment:
The link on this tag should be like where you want your users need to go to know more about your mod, not the download link.